base_app.htm

```diff
@@ -0,0 +1,43 @@
+## this is the skeleton of all pages on in this app - it defines the basic html tags
+<!DOCTYPE html>
+<%inherit file="/homepage/templates/base.htm" />
+<%! from catalog import models as cmod %>
+<html>
+    <meta charset="UTF-8">
+    <head>
+
+    </head>
+    <body>
+
+        <header>
+        </header>
+
+        <main>
+          <%block name="left">
```

Comment:
I might be a little OCD, but I typically like to indent different tags based on a parent relationship (inner tags indented in another time). This will help made the code easier to read, and will help you avoid different indenting problems that might come since python uses indenting to group code together.

Reply:
Ah yes! This is a good idea and for sure something I should go through my code to fix to increase readability and avoid problems in the future. Thank you!